src/main/java/com/faforever/client/rating/JSkillsRatingService.java

```diff
@@ -38,6 +38,6 @@ public double calculateQuality(Replay replay) {
           ));
           return team;
         })
-        .collect(Collectors.toList()));
+        .collect(Collectors.toList())) * 100;
```

Comment:
Since this is an internal, technical value, we keep it 0.0 - 1.0. To display, this can be multiplied by 100.